app/services/api/v3/finance/statements/index.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    module Finance
+      module Statements
+        class Index
+          attr_reader :cpd_lead_provider, :params
+
+          def initialize(cpd_lead_provider:, params:)
```

Comment:
Do we really need keyword arguments for only to arguments? I'd argue that up to 2 arguments it's not necessarily useful.
Another point, I'd rather have to params have passed to `def statements(filter)` so that you could apply different filters to the same instance, debugging in the console.

Reply:
I think using keyword arguments, generally makes our code more explicit, and adds a lot more flexibility to extending classes in the future without having to rely on order and being able to set defaults etc. This service is still likely to change/be extended with further statement endpoints, so I prefer to define keyword arguments. If this was unlikely to ever change then yes I would tend to agree.

Re filter passed through to statements, this service will likely be extended to serve a single statement, I'll revisit this suggestion then to see how we it all fits in